### ISSUE TITLE
Option to record attribute and gear damage when pushing a roll

### ISSUE THREAD

**Issue by subourbonite** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#43]
It would be incredibly helpful if the roller were able to offer to (or automatically, based on a setting) apply attribute or gear condition damage when a player pushes a roll and there are failures.

**Comment by aMediocreDad** [comment]
We are eventually switching from the Forbidden Lands roller to a roller @Stefouch built. When this happens I think we can look at building in a context menu that lets you apply pushed damage to yourself.

**Comment by aMediocreDad** [comment]
This is now being worked on.

As work progresses two things have become apparent. I need to refactor how failures are counted as current implementation counts only the total amount of skulls rolled, not discerning between gear and attributes.

I can only infer gear damage to weapons reliably since I cannot know what gear was used when doing other types of rolls. Armor also requires knowing whether the actor took damage.

Alternately I could show a button regardless and present a dialogue which asks what gear to deduct points from if there is no recorded gear in the roll.

**Comment by aMediocreDad** [comment]
Work on this issue has to continue after next release. 

The main problem with implementing this is that the current dice roller does not discern between rolled skulls on base dice and rolled skulls on gear dice. Thus there is need for a more extensive rework of the dice roller (this is already in the cards, see the Project Board).

**Comment by Stefouch** [comment]
> does not discern between rolled skulls on base dice and rolled skulls on gear dice
Is this an issue related to the YZUR ?

**Comment by aMediocreDad** [comment]
> Is this an issue related to the YZUR ?

No, this will be solved when moving to YZUR. It's the current dice roller that lacks the functionality.

Before implementing YZUR I need to clean up roll dialogs and roll handling as well as figure out how best to do spell/consumable rolls. 

There's also the DsN issue.
